.github/workflows/security-scan.yml

```diff
@@ -19,12 +19,12 @@ jobs:
       contents: read
       security-events: write # Needed to upload the results to code-scanning dashboard
     steps:
-      - name: Checkout code

```

Comment:
We deleted `Checkout code` in 2 places in this file was it not needed or mistake?